packages/fast-components-react-base/src/listbox/listbox.tsx

```diff
@@ -0,0 +1,314 @@
+import * as ReactDOM from "react-dom";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import { ListboxClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import {
+    ListboxHandledProps,
+    ListboxProps,
+    ListboxUnhandledProps,
+} from "./listbox.props";
+import * as React from "react";
+import { KeyCodes } from "@microsoft/fast-web-utilities";
+import { get, inRange, invert } from "lodash-es";
+import { canUseDOM } from "exenv-es6";
+
+export interface ListboxState {
+    /**
+     * The index of the focusable child
+     */
+    focusIndex: number;
+}
+
+export interface TypeAheadDataItem {
+    index: number;
+    compareString: string;
+}
+
+class Listbox extends Foundation<
```

Comment:
+1

Reply:
I am totally indifferent so happy to go with consensus.  I note that "listbox" exists as official markup as in:

role="listbox"

So that tells me that our lowercase title should be "listbox" and not "listBox".  given that I thought the capitalized version should be "Listbox" over "ListBox".   That being said I considered it a best guess and am happy to switch.

Reply:
So after some office conversation yesterday it seems there is no definitely right way to do this.  Current consensus is to stick with "Listbox" mostly because it is how we do "Checkbox".  Anyone who wants to argue otherwise please pile on.